src/utils/tabs.js

```diff
@@ -61,3 +61,51 @@ export function getSourceAnnotation(
     return <img className="blackBox" />;
   }
 }
+
+export function getTabMenuItems() {
+  const items = {
```

Comment:
maybe return { ... }

Reply:
Since it's always the same object you could skip the function and just

```js
export const tabMenuItems = {
...
};
```